src/app/component/user/components/profile/profile-dashboard/profile-dashboard.component.html

```diff
@@ -16,14 +16,14 @@
          [ngClass]="tabs.articles ? 'active' : 'plain'"
          (click)="toggleTab('articles')"
     >
-      <p>{{ 'profile.dashboard.my-articles' | translate }}</p>
+      <p>{{ 'profile.dashboard.my-tips-and-tricks' | translate }}</p>
     </div>
     <div class="hline"></div>
   </div>
   <div *ngIf="!loading; else spinner">
     <div class="in-progress header">
-      <p>{{ 'profile.dashboard.habits-in-progress' | translate }}
-        <span>{{ habitsInProgress.length }}</span>
+      <p>{{ 'profile.dashboard.habits-inprogress' | translate }} <br>
+        <span>{{ habitsInProgress.length }} {{ 'profile.dashboard.habits' | translate }}</span>
```

Comment:
maybe *ngIf="habitsInProgress" can be above {{ habitsInProgress.length }}

Reply:
I don't understand how to implement this correctly....

Reply:
Also, I agree with you about exequteRequests method that is in ngOnInit in profile-dashboard-component - it unnecessary, because it duplicates in method below